### ISSUE TITLE
AcousticBrainz API Integration?

### ISSUE THREAD

**Issue by hughrawlinson** [maxkueng/node-musicbrainz#21]
Hey,

Would it be interesting to potentially integrate the [AcousticBrainz API](http://acousticbrainz.org/data) into this project, or do you see that as out of the scope?

Basic background on AcousticBrainz if you aren't familiar with it: Universitat Pompeu Fabreau and MusicBrainz have teamed up to make this kickass DB of features for a 20m song dataset, and that API uses MusicBrainz IDs as keys for their data.

Basically it'd be a cheap addition to this library, which would add a whole other dimension to it, but on the other hand if you see it as out of scope because it's a separate API, cool :)
